msgid ""
msgstr ""
"Project-Id-Version: enigma2-vix-core\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-03-18 10:28+0330\n"
"PO-Revision-Date: \n"
"Last-Translator: litvcom\n"
"Language-Team: Russian localization by Ednaz && litvcom\n"
"Language: ru_RU\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
"X-Poedit-SourceCharset: UTF-8\n"
"X-Generator: Poedit 2.4.2\n"

#: ../src/BackupManager.py:146 ../src/BackupManager.py:289
#: ../src/ImageManager.py:276 ../src/ImageManager.py:408
msgid "%a %e %b  %-H:%M"
msgstr ""

#: ../src/ImageManager.py:1445
#, python-format
msgid "%s downloads"
msgstr "%s загрузок"

#: ../src/MountManager.py:109
#, python-format
msgid "%s: "
msgstr ""

#: ../src/MountManager.py:391
msgid "/media/hdd"
msgstr ""

#: ../src/SwapManager.py:326
msgid "128 Mb"
msgstr "128 МБ"

#: ../src/SwapManager.py:326
msgid "16 Mb"
msgstr "16 МБ"

#: ../src/SwapManager.py:326
msgid "256 Mb"
msgstr "256 МБ"

#: ../src/SwapManager.py:326
msgid "32 Mb"
msgstr "32 МБ"

#: ../src/SwapManager.py:326
msgid "64 Mb"
msgstr "64 МБ"

#: ../src/SwapManager.py:326
msgid "8 Mb"
msgstr "8 МБ"

#: ../src/SwapManager.py:326
msgid "96 Mb"
msgstr "96 МБ"

#: ../src/BackupManager.py:501 ../src/RestoreWizard.py:274
msgid "A background update check is in progress, please try again."
msgstr "Идет проверка наличия обновлений в фоновом режиме. Пожалуйста, попробуйте позже."

#: ../src/ClientModeBox.py:491
msgid "About"
msgstr ""

#: ../src/SwapManager.py:131 ../src/SwapManager.py:267
msgid "Activate"
msgstr "Активировать"

#: ../src/SwapManager.py:129 ../src/SwapManager.py:263
msgid "Active"
msgstr "Активный"

#: ../src/SoftcamManager.py:95 ../src/SoftcamManager.py:159
msgid "Active:"
msgstr "Активный:"

#: ../src/IPKInstaller.py:131
msgid "Are you ready to install ?"
msgstr "Вы готовы к установке?"

#: ../src/Multibootmgr.py:86
#, python-format
msgid "Are you sure you want to delete image slot %s ?"
msgstr "Вы уверены, что хотите удалить слот имиджа %s?"

#: ../src/ImageManager.py:1569
msgid ""
"Are you sure you want to download this image:\n"
" "
msgstr ""
"Вы уверены, что хотите скачать этот имидж:\n"
" "

#: ../src/BackupManager.py:359
msgid ""
"Are you sure you want to restore this backup:\n"
" "
msgstr ""
"Вы уверены, что хотите восстановить эту резервную копию:\n"
" "

#: ../src/ClientModeBox.py:524
msgid "Authentication"
msgstr "Идентификация"

#: ../src/SoftcamManager.py:120 ../src/SwapManager.py:133
msgid "Autostart"
msgstr "Автозапуск"

#: ../src/RestoreWizard.py:162
msgid "Background update check is in progress, please try again."
msgstr "Идет проверка наличие обновлений в фоновом режиме. Пожалуйста, попробуйте позже."

#: ../src/ImageManager.py:890
msgid "Backing up eMMC partitions for USB flash ..."
msgstr "Копирование разделов eMMC для USB-флеш..."

#: ../src/ImageManager.py:894
msgid "Backing up eMMC partitions for USB flash..."
msgstr "Копирование разделов eMMC для USB-флеш..."

#: ../src/BackupManager.py:1219 ../src/BackupManager.py:1223
#: ../src/BackupManager.py:1235 ../src/BackupManager.py:1239
msgid "Backing up files..."
msgstr "Резервное копирование файлов..."

#: ../src/ImageManager.py:874 ../src/ImageManager.py:878
msgid "Backing up kernel..."
msgstr "Резервное копирование ядра..."

#: ../src/ImageManager.py:882 ../src/ImageManager.py:886
msgid "Backing up root file system..."
msgstr "Резервное копирование корневой файловой системы..."

#: ../src/ui.py:56
msgid "Backup Manager"
msgstr "Менеджер бекапов"

#: ../src/ImageManager.py:922
msgid "Backup complete."
msgstr "Резервное копирование завершено."

#: ../src/BackupManager.py:1243
msgid "Backup complete..."
msgstr "Резервное копирование завершено..."

#: ../src/ImageManager.py:438
msgid "Backup confirmation"
msgstr "Подтверждение резервного копирования"

#: ../src/BackupManager.py:1030
msgid "Backup created"
msgstr "Резервная копия создана"

#: ../src/BackupManager.py:1397
msgid "Backup failed - e. g. wrong backup destination or no space left on backup device."
msgstr "Ошибка резервного копирования! Неправильное место назначения или на устройстве резервного копирования не осталось места."

#: ../src/BackupManager.py:335
msgid ""
"Backup in progress,\n"
"Please wait for it to finish, before trying again."
msgstr ""
"Выполняется резервное копирование,\n"
"Пожалуйста, дождитесь завершения, прежде чем пытаться снова."

#: ../src/BackupManager.py:965 ../src/ImageManager.py:167
msgid "Backup location"
msgstr "Расположение бекапа"

#: ../src/BackupManager.py:171 ../src/BackupManager.py:304
#: ../src/BackupManager.py:319 ../src/BackupManager.py:370
#: ../src/BackupManager.py:387 ../src/BackupManager.py:1201
#: ../src/ImageManager.py:458
msgid "Backup manager"
msgstr "Менеджер бекапов"

#: ../src/ui.py:57
msgid "Backup/Flash/ReBoot system image."
msgstr "Копирование/Прошивка/Перезагрузка имиджа"

#: ../src/MountManager.py:437 ../src/BackupManager.py:725
#: ../src/BackupManager.py:820 ../src/BackupManager.py:931
#: ../src/ImageManager.py:152 ../src/SoftcamManager.py:345
#: ../src/ClientModeBox.py:164 ../src/ClientModeBox.py:488
#: ../src/ClientModeBox.py:589 ../src/Multibootmgr.py:37
msgid "Cancel"
msgstr "Отменить"

#: ../src/ClientModeBox.py:644
msgid "Cannot download data. Please check your configuration"
msgstr "Не могу загрузить данные. Пожалуйста, проверьте вашу конфигурацию"

#: ../src/MountManager.py:302
#, python-format
msgid "Cannot unmount partition '%s'.  Make sure this partition is not in use.  (SWAP, record/timeshift, etc.)"
msgstr "Невозможно отсоединить раздел '%s' Убедитесь, что он не используется для подкачки, записи или таймшифта."

#: ../src/IPKInstaller.py:95 ../src/IPKInstaller.py:99
msgid "Change location"
msgstr "Изменить расположение"

#: ../src/SoftcamManager.py:353
msgid "Check interval on minutes"
msgstr "Проверьте интервал в минутах "

#: ../src/ImageManager.py:866
msgid "Checking free ram.."
msgstr "Проверка свободной оперативной памяти.."

#: ../src/SoftcamManager.py:651
msgid "Checking softcams..."
msgstr "Проверка эмулятора..."

#: ../src/BackupManager.py:934
msgid "Choose IPK folder"
msgstr "Выбрать папку с IPK"

#: ../src/BackupManager.py:933
msgid "Choose files"
msgstr "Выбрать файлы"

#: ../src/MountManager.py:428
msgid "Choose where to mount your devices to:"
msgstr "Выберите, точки монтирования устройств:"

#: ../src/ClientModeBox.py:590
msgid "Choose your main device"
msgstr "Выберите ваше основное устройство"

#: ../src/ui.py:62
msgid "Client Mode Box"
msgstr "Окно режима клиента"

#: ../src/ClientModeBox.py:895
msgid "Client Mode Box: If you want to exit Client Mode and have a backup with your original settings. You can restore from blue button on Vision Backup Manager."
msgstr "Окно режима клиента: если вы хотите выйти из режима клиента и создать резервную копию с исходными настройками. Вы можете выполнить восстановление с помощью синей кнопки в Vision менеджере бекапов."

#: ../src/ui.py:67 ../src/ImageManager.py:1448 ../src/ImageManager.py:1457
#: ../src/SoftcamManager.py:117 ../src/IPKInstaller.py:61
#: ../src/IPKInstaller.py:184 ../src/SwapManager.py:130
msgid "Close"
msgstr "Закрыть"

#: ../src/BackupManager.py:409 ../src/BackupManager.py:413
msgid "Comparing against backup..."
msgstr "Сравнение с резервной копией..."

#: ../src/ImageManager.py:1537
msgid "Compress"
msgstr "Сжатие"

#: ../src/SwapManager.py:132 ../src/SwapManager.py:196
msgid "Create"
msgstr "Создать"

#: ../src/ui.py:61
msgid "Create and Manage your SWAP files."
msgstr "Создание и управление файлом подкачки"

#: ../src/ImageManager.py:1070 ../src/ImageManager.py:1073
#: ../src/ImageManager.py:1076 ../src/ImageManager.py:1078
#: ../src/ImageManager.py:1080
msgid "Create:"
msgstr "Создать:"

#: ../src/BackupManager.py:401 ../src/BackupManager.py:405
#: ../src/BackupManager.py:1211 ../src/BackupManager.py:1215
msgid "Creating list of installed plugins..."
msgstr "Создание списка установленных плагинов..."

#: ../src/ImageManager.py:870
msgid "Creating swap.."
msgstr "Создание файла подкачки.."

#: ../src/ImageManager.py:914 ../src/ImageManager.py:918
msgid "Creating zip..."
msgstr "Создание ZIP архива..."

#: ../src/SwapManager.py:156 ../src/SwapManager.py:263
#: ../src/SwapManager.py:268
msgid "Current status:"
msgstr "Текущий статус:"

#: ../src/BackupManager.py:61 ../src/ImageManager.py:73
#: ../src/ClientModeBox.py:52
msgid "Daily"
msgstr "Ежедневно"

#: ../src/SwapManager.py:262
msgid "Deactivate"
msgstr "Отключить"

#: ../src/ClientModeBox.py:526
msgid "Default username (root)"
msgstr "Имя пользователя по умолчанию (root)"

#: ../src/BackupManager.py:127 ../src/ImageManager.py:245
#: ../src/SwapManager.py:206 ../src/SwapManager.py:217
msgid "Delete"
msgstr "Удалить"

#: ../src/BackupManager.py:759
msgid "Deselect"
msgstr "Отменить выбор"

#: ../src/MountManager.py:97
#, python-format
msgid "Device %s"
msgstr "Устройство %s"

#: ../src/MountManager.py:152 ../src/MountManager.py:296
#: ../src/MountManager.py:312 ../src/MountManager.py:326
#: ../src/MountManager.py:359 ../src/MountManager.py:372
#: ../src/BackupManager.py:223 ../src/BackupManager.py:226
#: ../src/BackupManager.py:252 ../src/ImageManager.py:366
#: ../src/ImageManager.py:376
msgid "Device: "
msgstr "Устройство: "

#: ../src/BackupManager.py:202
msgid "Device: None available"
msgstr "Устройство: не доступно"

#: ../src/ImageManager.py:347
msgid "Device: Un-mount"
msgstr "Устройство: Размонтировать"

#: ../src/SoftcamManager.py:155
msgid "Disable startup"
msgstr "Отключить запуск"

#: ../src/ImageManager.py:436
msgid ""
"Do you want to create a full image backup?\n"
"This can take about 6 minutes to complete."
msgstr ""
"Вы хотите создать полную резервную копию имиджа?\n"
"Это может занять около 6 минут."

#: ../src/ImageManager.py:478
#, python-format
msgid ""
"Do you want to flash image\n"
"%s"
msgstr ""
"Вы хотите прошить имидж\n"
"%s"

#: ../src/IPKInstaller.py:144
msgid "Do you want to restart GUI now ?"
msgstr "Хотите перезагрузить GUI сейчас?"

#: ../src/BackupManager.py:656
msgid "Do you want to restore your Enigma2 plugins ?"
msgstr "Хотите восстановить свои плагины Enigma2?"

#: ../src/BackupManager.py:437
msgid "Do you want to restore your enigma2 settings ?"
msgstr "Хотите восстановить свои настройки Enigma2?"

#: ../src/ClientModeBox.py:513
msgid "Do you want to scan for a server?"
msgstr "Хотите перезагрузить GUI сейчас?"

#: ../src/ImageManager.py:1539
msgid "DownLoad"
msgstr "Загрузить"

#: ../src/ImageManager.py:1449 ../src/ImageManager.py:1458
msgid "Download"
msgstr "Загрузить"

#: ../src/ClientModeBox.py:640
msgid "Download completed"
msgstr "Загрузка завершена"

#: ../src/ImageManager.py:1571
msgid "Download confirmation"
msgstr "Подтверждение загрузки"

#: ../src/ImageManager.py:1586
msgid "Downloading"
msgstr "Загрузка"

#: ../src/ImageManager.py:247
msgid "Downloads"
msgstr "Загрузить"

#: ../src/ImageManager.py:547
msgid ""
"ET8500 Multiboot: Yes to restore OS1\n"
"No to restore OS2:\n"
" "
msgstr ""
"Мультизагрузка: Да, чтобы восстановить ОС1\n"
"Нет, чтобы восстановить ОС2:\n"
" "

#: ../src/ImageManager.py:549
msgid "ET8500 image restore"
msgstr "Восстановление имиджа"

#: ../src/Multibootmgr.py:73
msgid "Empty slot"
msgstr "Пустой слот"

#: ../src/SoftcamManager.py:351
msgid "Enable frozen check"
msgstr "Включить проверку на зависание"

#: ../src/SoftcamManager.py:157
msgid "Enable startup"
msgstr "Включить запуск"

#: ../src/SwapManager.py:270
msgid "Enable swap at startup"
msgstr "Файл подкачки при загрузке"

#: ../src/BackupManager.py:1164 ../src/ImageManager.py:741
msgid "Enough retries, delaying till next schedule."
msgstr "Достаточно повторных попыток, подождите до следующего старта."

#: ../src/Multibootmgr.py:40
msgid "Erase"
msgstr "Очистить"

#: ../src/RestoreWizard.py:96
msgid "Exit the restore wizard"
msgstr "Выход из мастера восстановления"

#: ../src/ImageManager.py:1537
msgid "Expand"
msgstr "Расширить"

#: ../src/IPKInstaller.py:90 ../src/IPKInstaller.py:109
msgid "Extra IPK's"
msgstr "Дополнительные IPK"

#: ../src/ImageManager.py:248
msgid "Flash"
msgstr "Прошить"

#: ../src/ImageManager.py:545
msgid "Flash image unzip successful."
msgstr "Имидж успешно распакован."

#: ../src/ImageManager.py:623
msgid "Flash on slot successful"
msgstr "Имидж успешно прошит."

#: ../src/BackupManager.py:966 ../src/BackupManager.py:985
#: ../src/ImageManager.py:168 ../src/ImageManager.py:187
msgid "Folder prefix"
msgstr "Префикс папки"

#: ../src/SoftcamManager.py:245 ../src/SoftcamManager.py:299
msgid "Found non-standard softcam, trying to start, this may fail."
msgstr "Найден неподдерживаемый эмулятор, попытка запуска, но может не удаться."

#: ../src/ImageManager.py:366
msgid "Free space:"
msgstr "Свободно памяти:"

#: ../src/ImageManager.py:383
msgid "From which image library do you want to download?"
msgstr "Из какой библиотеки имиджей Вы хотите скачать?"

#: ../src/ui.py:65
msgid "H9 SDcard Manager"
msgstr "Zhemma H9 SDCard менеджер"

#: ../src/ClientModeBox.py:522
msgid "HTTP port"
msgstr "HTTP порт"

#: ../src/ClientModeBox.py:521
msgid "Host"
msgstr "Хост"

#: ../src/BackupManager.py:129 ../src/SoftcamManager.py:123
msgid "INFO"
msgstr "ИНФО"

#: ../src/IPKInstaller.py:177
msgid "IPK installer"
msgstr "IPK установщик"

#: ../src/ui.py:57
msgid "Image Manager"
msgstr "Имидж менеджер"

#: ../src/ImageManager.py:296 ../src/ImageManager.py:427
#: ../src/ImageManager.py:448 ../src/ImageManager.py:860
msgid "Image manager"
msgstr "Имидж менеджер"

#: ../src/ClientModeBox.py:487
msgid "Important: Do not enable OpenWebif authentication on server, neither in this Setup."
msgstr "Важно: не включайте аутентификацию OpenWebif ни на сервере, ни в этой настройке."

#: ../src/SwapManager.py:128 ../src/SwapManager.py:268
msgid "Inactive"
msgstr "Неактивен"

#: ../src/SwapManager.py:287 ../src/SwapManager.py:353
msgid "Info"
msgstr "Инфо"

#: ../src/H9SDmanager.py:37
msgid "Init SDcard"
msgstr "Инициализация SD-карты"

#: ../src/H9SDmanager.py:38
msgid "Init USB/SDA1"
msgstr "Инициализация usb/sda1"

#: ../src/H9SDmanager.py:50
msgid "Init Zgemma H9 SDcard - please reboot after use."
msgstr "Инициализация Zgemma H9 SD-карты - пожалуйста, перезагрузите после использования."

#: ../src/H9SDmanager.py:72
msgid "Init Zgemma H9 USB/SDA1 - please reboot after use."
msgstr "Инициализация Zgemma H9 usb/sda1 - пожалуйста, перезагрузите после использования."

#: ../src/IPKInstaller.py:62 ../src/IPKInstaller.py:185
msgid "Install"
msgstr "Установить"

#: ../src/ui.py:58
msgid "Install IPK's from your tmp folder."
msgstr "Установка IPK из директории tmp"

#: ../src/IPKInstaller.py:133
msgid "Install confirmation"
msgstr "Подтверждение установки"

#: ../src/plugin.py:187
msgid "Install extensions."
msgstr "Установить дополнения."

#: ../src/IPKInstaller.py:141
msgid "Installing..."
msgstr "Установка..."

#: ../src/IPKInstaller.py:187
msgid "Invert"
msgstr "Инвертировать"

#: ../src/IPKInstaller.py:93
msgid "It seems you have not setup an extra location. Please set it up in the Backup manager setup menu."
msgstr "Похоже, дополнительное местоположение не было настроено. Пожалуйста, настройте его в меню настроек резервного копирования."

#: ../src/ui.py:50 ../src/MountManager.py:218 ../src/MountManager.py:430
#: ../src/ScriptRunner.py:52 ../src/BackupManager.py:117
#: ../src/BackupManager.py:814 ../src/BackupManager.py:914
#: ../src/BackupManager.py:1040 ../src/ImageManager.py:131
#: ../src/ImageManager.py:251 ../src/ImageManager.py:1451
#: ../src/SoftcamManager.py:88 ../src/SoftcamManager.py:328
#: ../src/SoftcamManager.py:403 ../src/SoftcamManager.py:488
#: ../src/SoftcamManager.py:573 ../src/IPKInstaller.py:42
#: ../src/SwapManager.py:114 ../src/ClientModeBox.py:123
#: ../src/ClientModeBox.py:480 ../src/ClientModeBox.py:887
msgid "Lets define enigma2 once more"
msgstr "Давайте еще раз определим enigma2"

#: ../src/ImageManager.py:366
msgid "MB"
msgstr "МБ"

#: ../src/BackupManager.py:128 ../src/ImageManager.py:249
#: ../src/SoftcamManager.py:122
msgid "MENU"
msgstr "МЕНЮ"

#: ../src/ui.py:56
msgid "Manage settings backup."
msgstr "Управление резервными копиями"

#: ../src/ui.py:59
msgid "Manage your devices mount points."
msgstr "Управление точками монтирования своих устройств"

#: ../src/BackupManager.py:61 ../src/ImageManager.py:73
#: ../src/ClientModeBox.py:52
msgid "Monthly"
msgstr "Ежемесячно"

#: ../src/MountManager.py:227
msgid "Mount"
msgstr "Монтировать"

#: ../src/ui.py:59
msgid "Mount Manager"
msgstr "Менеджер монтирования"

#: ../src/MountManager.py:315
#, python-format
msgid "Mount failed for '%s', error code = '%s'."
msgstr "Не удалось смонтировать '%s', код ошибки = '%s'."

#: ../src/MountManager.py:152 ../src/MountManager.py:254
#: ../src/MountManager.py:295 ../src/MountManager.py:311
#: ../src/MountManager.py:371
msgid "Mount: "
msgstr "Монтировать: "

#: ../src/ui.py:65
msgid "Move Nand root to SD card"
msgstr "Переместить корень NAND на SD-карту"

#: ../src/ImageManager.py:906 ../src/ImageManager.py:910
msgid "Moving to backup Location..."
msgstr "Перемещение в каталог резервного копирования..."

#: ../src/ImageManager.py:553 ../src/ImageManager.py:1347
msgid "Multiboot only able to restore this backup to MMC slot1"
msgstr "Мультизагрузчик может восстановить эту резервную копию только в слот1 MMC"

#: ../src/BackupManager.py:125 ../src/BackupManager.py:176
#: ../src/ImageManager.py:301
msgid "New backup"
msgstr "Новая копия"

#: ../src/BackupManager.py:146 ../src/BackupManager.py:148
#: ../src/BackupManager.py:289 ../src/BackupManager.py:291
#: ../src/ImageManager.py:276 ../src/ImageManager.py:278
#: ../src/ImageManager.py:408 ../src/ImageManager.py:410
msgid "Next backup: "
msgstr "Следующая копия: "

#: ../src/SoftcamManager.py:241
msgid ""
"No config files found, please setup Ncam first\n"
"in /etc/tuxbox/config/ncam"
msgstr ""
"Конфигурационных файлов не найдено, сначала настройте Ncam\n"
"в /etc/tuxbox/config/ncam"

#: ../src/SoftcamManager.py:289
msgid ""
"Конфигурационных файлов не найдено, сначала настройте Ncam\n"
"in /etc/tuxbox/config/ncam."
msgstr ""
"Конфигурационные файлы не найдены, настройте Ncam\n"
"в /etc/tuxbox/config/ncam."

#: ../src/SoftcamManager.py:226
msgid ""
"No config files found, please setup Oscam first\n"
"in /etc/tuxbox/config/oscam"
msgstr ""
"Конфигурационные файлы не найдены, настройте OSCam\n"
"в /etc/tuxbox/config/oscam"

#: ../src/SoftcamManager.py:285
msgid ""
"No config files found, please setup Oscam first\n"
"in /etc/tuxbox/config/oscam."
msgstr ""
"Конфигурационные файлы не найдены, настройте OSCam\n"
"в /etc/tuxbox/config/oscam."

#: ../src/SoftcamManager.py:236
msgid ""
"No config files found, please setup Oscam-emu first\n"
"in /etc/tuxbox/config/oscam-emu"
msgstr ""
"Конфигурационные файлы не найдены, настройте Oscam-emu\n"
"в /etc/tuxbox/config/oscam-emu"

#: ../src/SoftcamManager.py:293
msgid ""
"No config files found, please setup Oscam-emu first\n"
"in /etc/tuxbox/config/oscam-emu."
msgstr ""
"Конфигурационные файлы не найдены, настройте Oscam-emu\n"
"в /etc/tuxbox/config/oscam-emu."

#: ../src/SoftcamManager.py:231
msgid ""
"No config files found, please setup Oscam-smod first\n"
"in /etc/tuxbox/config/oscam-smod"
msgstr ""
"Конфигурационные файлы не найдены, настройте Oscam-smod\n"
"в /etc/tuxbox/config/oscam-smod"

#: ../src/SoftcamManager.py:297
msgid ""
"No config files found, please setup Oscam-smod first\n"
"in /etc/tuxbox/config/oscam-smod."
msgstr ""
"Конфигурационные файлы не найдены, настройте Oscam-smod\n"
"в /etc/tuxbox/config/oscam-smod."

#: ../src/ClientModeBox.py:593
msgid "No devices found"
msgstr "Устройства не найдены"

#: ../src/ImageManager.py:1477
msgid "No images found on the selected download server...if password check validity"
msgstr "На выбранном сервере не найдено ни одного имиджа. Проверьте с паролем"

#: ../src/RestoreWizard.py:107 ../src/RestoreWizard.py:110
msgid "No, do not restore plugins"
msgstr "Нет, не восстанавливать плагины"

#: ../src/RestoreWizard.py:100
msgid "No, do not restore settings"
msgstr "Нет, не восстанавливать настройки"

#: ../src/MountManager.py:118 ../src/MountManager.py:120
#: ../src/MountManager.py:131 ../src/MountManager.py:392
msgid "None"
msgstr "Нет"

#: ../src/Multibootmgr.py:39
msgid "Note: slot list does not show current image or empty slots."
msgstr "Информация: в списке слотов не отображается текущие имиджи или слоты пусты."

#: ../src/BackupManager.py:673
msgid "Now skipping restore process"
msgstr "Пропустить восстановление"

#: ../src/BackupManager.py:932 ../src/ImageManager.py:153
#: ../src/RestoreWizard.py:114
msgid "OK"
msgstr "OK"

#: ../src/RestoreWizard.py:96
msgid "OK, to perform a restore"
msgstr "ОК, чтобы восстановить"

#: ../src/ui.py:49 ../src/MountManager.py:217 ../src/MountManager.py:429
#: ../src/ScriptRunner.py:51 ../src/BackupManager.py:116
#: ../src/BackupManager.py:813 ../src/BackupManager.py:913
#: ../src/BackupManager.py:1039 ../src/ImageManager.py:130
#: ../src/ImageManager.py:250 ../src/ImageManager.py:1450
#: ../src/SoftcamManager.py:87 ../src/SoftcamManager.py:327
#: ../src/SoftcamManager.py:402 ../src/SoftcamManager.py:487
#: ../src/SoftcamManager.py:572 ../src/IPKInstaller.py:41
#: ../src/SwapManager.py:113 ../src/ClientModeBox.py:122
#: ../src/ClientModeBox.py:479 ../src/ClientModeBox.py:886
msgid "OpenVision"
msgstr ""

#: ../src/plugin.py:205
msgid "Opkg"
msgstr ""

#: ../src/ui.py:58
msgid "Opkg Install"
msgstr "Установить"

#: ../src/MountManager.py:299
#, python-format
msgid "Partition: %s  Mount: %s unmounted successfully; if all partitions now unmounted you can remove device."
msgstr "Раздел: %s Монтирование: %s успешно размонтировано; если все разделы отключены, вы можете удалить устройство."

#: ../src/ClientModeBox.py:527
msgid "Password"
msgstr "Пароль"

#: ../src/BackupManager.py:876
msgid "Please enter a folder that contains some packages."
msgstr "Пожалуйста, войдите в папку, содержащую несколько пакетов."

#: ../src/SwapManager.py:319
msgid "Please select device to use as SWAP file location."
msgstr "Выберите устройство, которое будет использоваться в качестве расположения SWAP файла."

#: ../src/ClientModeBox.py:557
msgid ""
"Please wait while download is in progress.\n"
"Note: If you have parental control enabled on remote box, the local settings will be overwritten."
msgstr ""
"Подождите, пока идет загрузка.\n"
"Примечание: Если на удаленном компьютере включен родительский контроль, локальные настройки будут перезаписаны."

#: ../src/RestoreWizard.py:146
msgid "Please wait while plugins restore completes..."
msgstr "Пожалуйста, подождите, пока завершится восстановление плагинов..."

#: ../src/ClientModeBox.py:569
msgid ""
"Please wait while scan is in progress.\n"
"This operation may take a while"
msgstr ""
"Подождите, пока идет сканирование.\n"
"Эта операция может занять некоторое время."

#: ../src/MountManager.py:228
msgid "Please wait while scanning for devices..."
msgstr "Пожалуйста, подождите, пока идет сканирование для устройств..."

#: ../src/MountManager.py:450
msgid "Please wait while scanning your receiver devices..."
msgstr "Пожалуйста, подождите, пока просканируют ваши приемные устройства..."

#: ../src/RestoreWizard.py:201
msgid "Please wait while settings restore completes..."
msgstr "Пожалуйста, подождите, пока завершится восстановление настроек..."

#: ../src/SoftcamManager.py:409
msgid "Please wait while starting\n"
msgstr "Подождите, идет запуск\n"

#: ../src/SoftcamManager.py:494
msgid "Please wait while stopping\n"
msgstr "Подождите, идет остановка\n"

#: ../src/ImageManager.py:533
msgid "Please wait while the flash prepares."
msgstr "Пожалуйста, подождите, пока подготавливается прошивка."

#: ../src/RestoreWizard.py:134 ../src/RestoreWizard.py:139
msgid "Please wait while the system gathers information..."
msgstr "Пожалуйста, подождите, пока система соберет информацию..."

#: ../src/MountManager.py:472
msgid "Please wait."
msgstr "Ждите пожалуйста."

#: ../src/BackupManager.py:1335
msgid "Plugin listing failed - e. g. wrong backup destination or no space left on backup device."
msgstr "Не удалось добавить плагин! Неправильное место назначения или на устройстве резервного копирования не осталось места."

#: ../src/BackupManager.py:1227 ../src/BackupManager.py:1231
msgid "Preparing extra plugins..."
msgstr "Подготовка дополнительных плагинов..."

#: ../src/IPKInstaller.py:188
msgid "Press OK to toggle the selection."
msgstr "Нажмите ОК для подтверждения выбора или отмены."

#: ../src/H9SDmanager.py:35
msgid "Press appropiate Init to move Nand root to SDcard or USB."
msgstr "Нажмите 'Инициализация', чтобы переместить корень NAND на SD-карту."

#: ../src/BackupManager.py:1013 ../src/ImageManager.py:215
msgid "Really close without saving settings?"
msgstr "Закрыть без сохранения настроек?"

#: ../src/MountManager.py:329
#, python-format
msgid "Really use and mount %s as HDD ?"
msgstr "Действительно использовать и монтировать %s в качестве HDD ?"

#: ../src/H9SDmanager.py:36
msgid "Reboot"
msgstr "Перезагрузка..."

#: ../src/BackupManager.py:429
msgid "Rebooting..."
msgstr "Перезагрузка..."

#: ../src/ImageManager.py:476
#, python-format
msgid ""
"Recording(s) are in progress or coming up in few seconds!\n"
"Do you still want to flash image\n"
"%s?"
msgstr ""
"Запись(и) идёт или начнётся в ближайшие несколько секунд!\n"
"Вы все еще хотите прошить имидж\n"
"%s?"

#: ../src/Multibootmgr.py:82
#, python-format
msgid "Removal of this slot will not show in %s GUI.  Are you sure you want to delete image slot?"
msgstr "Удаление этого слота не будет отображаться в %s GUI. Вы уверены, что хотите удалить слот %s ?"

#: ../src/Multibootmgr.py:84 ../src/Multibootmgr.py:88
msgid "Remove confirmation"
msgstr "Удаление с подтверждением"

#: ../src/ImageManager.py:898 ../src/ImageManager.py:902
msgid "Removing temp mounts..."
msgstr "Удаление временных монтирований..."

#: ../src/BackupManager.py:970 ../src/ImageManager.py:172
#: ../src/ClientModeBox.py:533
msgid "Repeat how often"
msgstr "Как часто повторять"

#: ../src/ui.py:51 ../src/MountManager.py:219 ../src/MountManager.py:431
#: ../src/ScriptRunner.py:53 ../src/BackupManager.py:118
#: ../src/BackupManager.py:815 ../src/BackupManager.py:915
#: ../src/BackupManager.py:1041 ../src/ImageManager.py:132
#: ../src/ImageManager.py:252 ../src/ImageManager.py:1452
#: ../src/SoftcamManager.py:89 ../src/SoftcamManager.py:329
#: ../src/SoftcamManager.py:404 ../src/SoftcamManager.py:489
#: ../src/SoftcamManager.py:574 ../src/IPKInstaller.py:43
#: ../src/SwapManager.py:115 ../src/ClientModeBox.py:124
#: ../src/ClientModeBox.py:481 ../src/ClientModeBox.py:888
msgid "Report problems to:"
msgstr "Сообщайте о проблемах по адресу:"

#: ../src/SoftcamManager.py:152
msgid "Restart"
msgstr "Перезагрузить"

#: ../src/IPKInstaller.py:146
msgid "Restart GUI."
msgstr "Перезагрузка GUI."

#: ../src/MountManager.py:404 ../src/MountManager.py:477
msgid "Restart receiver."
msgstr "Перезагрузка ресивера."

#: ../src/BackupManager.py:126
msgid "Restore"
msgstr "Восстановить"

#: ../src/BackupManager.py:361
msgid "Restore confirmation"
msgstr "Подтверждение восстановления"

#: ../src/BackupManager.py:341
msgid "Restore only settings from this backup ?\n"
msgstr "Восстановить только настройки из этой резервной копии ?\n"

#: ../src/BackupManager.py:130
msgid "Restore settings"
msgstr "Восстановить настройки"

#: ../src/RestoreWizard.py:135 ../src/RestoreWizard.py:140
#: ../src/RestoreWizard.py:147 ../src/RestoreWizard.py:153
#: ../src/RestoreWizard.py:159 ../src/RestoreWizard.py:163
#: ../src/RestoreWizard.py:189 ../src/RestoreWizard.py:193
#: ../src/RestoreWizard.py:202 ../src/RestoreWizard.py:259
#: ../src/RestoreWizard.py:263 ../src/RestoreWizard.py:267
#: ../src/RestoreWizard.py:271 ../src/RestoreWizard.py:275 ../src/plugin.py:204
msgid "Restore wizard"
msgstr "Мастер восстановления"

#: ../src/BackupManager.py:389 ../src/BackupManager.py:393
#: ../src/BackupManager.py:397
msgid "Restoring backup..."
msgstr "Восстановление резервной копии..."

#: ../src/BackupManager.py:703
msgid "Restoring please wait..."
msgstr "Восстановление, подождите ..."

#: ../src/BackupManager.py:425
msgid "Restoring plugins, this can take a long time..."
msgstr "Восстановление плагинов, это может занять много времени..."

#: ../src/BackupManager.py:417 ../src/BackupManager.py:421
msgid "Restoring plugins..."
msgstr "Восстановление плагинов..."

#: ../src/BackupManager.py:453
msgid "Restoring, your receiver go to restart..."
msgstr "Восстанавливаю, ваш ресивер перезагрузится..."

#: ../src/Multibootmgr.py:42
msgid "Retrieving image slots - Please wait..."
msgstr "Получение загрузочных разделов. Пожалуйста, подождите..."

#: ../src/ScriptRunner.py:58
msgid "Run"
msgstr "Запустить"

#: ../src/ui.py:60
msgid "Run your shell scripts."
msgstr "Запуск своих скриптов"

#: ../src/MountManager.py:436 ../src/BackupManager.py:726
#: ../src/BackupManager.py:821 ../src/SoftcamManager.py:346
#: ../src/ClientModeBox.py:489
msgid "Save"
msgstr "Сохранить"

#: ../src/ClientModeBox.py:490
msgid "Scan"
msgstr "Сканирование"

#: ../src/BackupManager.py:967 ../src/BackupManager.py:975
#: ../src/ImageManager.py:169 ../src/ImageManager.py:177
msgid "Schedule backups"
msgstr "Запланировать резервное копирование"

#: ../src/ClientModeBox.py:530
msgid "Schedule sync"
msgstr "Расписание синхронизации"

#: ../src/ui.py:60
msgid "Script Runner"
msgstr "Запуск скриптов"

#: ../src/BackupManager.py:761
msgid "Select"
msgstr "Выбрать"

#: ../src/BackupManager.py:219 ../src/BackupManager.py:223
#: ../src/BackupManager.py:226
msgid "Select a backup to restore:"
msgstr "Выберите резервную копию для восстановления:"

#: ../src/IPKInstaller.py:113
msgid "Select a package to install:"
msgstr "Выберите пакет для установки:"

#: ../src/ImageManager.py:1456
msgid "Select an image to download:"
msgstr "Выберите имидж для загрузки:"

#: ../src/BackupManager.py:812
msgid "Select extra packages folder"
msgstr "Пожалуйста, выберите папку с плагинами"

#: ../src/BackupManager.py:723
msgid "Select files/folders to backup"
msgstr "Выбор файлов/папок для резервного копирования"

#: ../src/BackupManager.py:850
msgid "Select folder that contains plugins"
msgstr "Выберите папку, содержащую плагины"

#: ../src/SwapManager.py:327
msgid "Select the swap file size:"
msgstr "Выберите размер файла подкачки:"

#: ../src/ImageManager.py:366
msgid "Select what you want to do"
msgstr "Выберите то, что вы хотите сделать"

#: ../src/SoftcamManager.py:94
msgid "Select:"
msgstr "Выбрать:"

#: ../src/ImageManager.py:862
msgid "Setting up..."
msgstr "Настройка..."

#: ../src/MountManager.py:225
msgid "Setup mounts"
msgstr "Настройки монтирования"

#: ../src/MountManager.py:132
msgid "Size: "
msgstr "Размер: "

#: ../src/MountManager.py:141
#, python-format
msgid "Size: %sGB"
msgstr "Размер: %sGB"

#: ../src/MountManager.py:139
#, python-format
msgid "Size: %sMB"
msgstr "Размер: %sMB"

#: ../src/MountManager.py:143
#, python-format
msgid "Size: %sTB"
msgstr "Размер: %sTB"

#: ../src/MountManager.py:137
msgid "Size: unavailable"
msgstr "Размер: недоступно"

#: ../src/SoftcamManager.py:401
msgid "Softcam starting..."
msgstr "Запуск эмулятора..."

#: ../src/SoftcamManager.py:486
msgid "Softcam stopping..."
msgstr "Остановка эмулятора..."

#: ../src/SoftcamManager.py:649
msgid "SoftcamCheck"
msgstr "Проверка эмулятора"

#: ../src/IPKInstaller.py:97
msgid "Sorry but that location does not exist or is not setup. Please set it up in the Backup manager setup menu."
msgstr "Извините, но это местоположение не существует или не настроено. Пожалуйста, настройте его в меню настроек резервного копирования."

#: ../src/BackupManager.py:549
msgid "Sorry the feeds are down for maintenance. Please try again later."
msgstr "Извините, сервер отключен для обслуживания, повторите попытку позже."

#: ../src/RestoreWizard.py:152
msgid "Sorry the feeds are down for maintenance. Please try using Backup manager to restore plugins later."
msgstr "Извините, сервер отключен для обслуживания. Для восстановления плагинов, повторите попытку позже."

#: ../src/RestoreWizard.py:270
msgid "Sorry the feeds are down for maintenance. Please try using the Backup manager to restore plugins later."
msgstr "Извините, сервер отключен для обслуживания. Для восстановления плагинов, повторите попытку позже."

#: ../src/RestoreWizard.py:188 ../src/RestoreWizard.py:192
msgid "Sorry, but the file is not compatible with this image version."
msgstr "Извините, но файл не совместим с этой версией имиджа."

#: ../src/BackupManager.py:354 ../src/BackupManager.py:356
msgid "Sorry, but the file is not compatible with this kernel version."
msgstr "Извините, но файл несовместим с этой версией имиджа."

#: ../src/BackupManager.py:473
msgid "Sorry, but the restore failed."
msgstr "Извините, но восстановление не удалось."

#: ../src/SwapManager.py:321
msgid "Sorry, no physical devices that supports SWAP attached. Can't create SWAP file on network or fat32 file-systems."
msgstr "К сожалению, нет устройств, поддерживающих файл подкачки. Невозможно создать подкачку в сетевых или файловых системах FAT32."

#: ../src/ImageManager.py:982
msgid "Sorry, not enough free RAM found, and no physical devices that supports swap attached. Can't create SWAP file on network or FAT32 file-systems, unable to make backup."
msgstr "К сожалению, недостаточно свободной оперативной памяти, и нет устройств, поддерживающих файл подкачки. Невозможно создать резервную копию!"

#: ../src/ui.py:53 ../src/MountManager.py:221 ../src/MountManager.py:433
#: ../src/ScriptRunner.py:55 ../src/BackupManager.py:120
#: ../src/BackupManager.py:817 ../src/BackupManager.py:917
#: ../src/BackupManager.py:1043 ../src/ImageManager.py:134
#: ../src/ImageManager.py:254 ../src/ImageManager.py:1454
#: ../src/SoftcamManager.py:91 ../src/SoftcamManager.py:331
#: ../src/SoftcamManager.py:406 ../src/SoftcamManager.py:491
#: ../src/SoftcamManager.py:576 ../src/IPKInstaller.py:45
#: ../src/SwapManager.py:117 ../src/ClientModeBox.py:126
#: ../src/ClientModeBox.py:483 ../src/ClientModeBox.py:890
msgid "Sources are available at:"
msgstr "Исходники доступны по адресу:"

#: ../src/SoftcamManager.py:146
msgid "Start"
msgstr "Старт"

#: ../src/BackupManager.py:1203 ../src/BackupManager.py:1207
msgid "Starting..."
msgstr "Запуск..."

#: ../src/SwapManager.py:127
msgid "Status:"
msgstr "Статус:"

#: ../src/SoftcamManager.py:148
msgid "Stop"
msgstr "Стоп"

#: ../src/ClientModeBox.py:523
msgid "Streaming port"
msgstr "Порт стриминга"

#: ../src/ui.py:61
msgid "Swap Manager"
msgstr "Управление файлом подкачки"

#: ../src/SwapManager.py:286
msgid "Swap file not found. You have to create the file before you try to activate it."
msgstr "Файл подкачки не найден. Вы должны создать файл, прежде чем пытаться его активировать."

#: ../src/SwapManager.py:123
msgid "Swap place:"
msgstr "Путь:"

#: ../src/SwapManager.py:125
msgid "Swap size:"
msgstr "Размер файла подкачки:"

#: ../src/IPKInstaller.py:103 ../src/IPKInstaller.py:111
msgid "Temp folder"
msgstr "Временная папка"

#: ../src/BackupManager.py:1298
msgid "The backup location does not have enough free space."
msgstr "В хранилище резервных копий недостаточно свободного места."

#: ../src/ImageManager.py:944
msgid "The backup location does not have enough free space.\n"
msgstr "В хранилище резервных копий недостаточно свободного места.\n"

#: ../src/MountManager.py:402 ../src/MountManager.py:475
msgid ""
"The changes need a system restart to take effect.\n"
"Restart your receiver now?"
msgstr ""
"Чтобы изменения вступили в силу, необходимо перезагрузить систему.\n"
"Перезагрузить Ваш ресивер сейчас?"

#: ../src/BackupManager.py:219
msgid "The chosen location does not exist, using."
msgstr "Выбранное местоположение не существует, используется /media/hdd."

#: ../src/BackupManager.py:252 ../src/ImageManager.py:376
msgid "There is a problem with this device. Please reformat it and try again."
msgstr "Есть проблема с этим устройством. Пожалуйста, отформатируйте и попробуйте снова."

#: ../src/BackupManager.py:333
msgid "There is no backup to restore."
msgstr "Нет резервной копии для восстановления."

#: ../src/ImageManager.py:528
msgid "There is no image to flash."
msgstr "Нет имиджа для прошивки."

#: ../src/MountManager.py:330 ../src/MountManager.py:376
msgid "This Device is already mounted as HDD."
msgstr "Это устройство уже было установлено как HDD."

#: ../src/BackupManager.py:969 ../src/ImageManager.py:171
msgid "Time of backup to start in minutes"
msgstr "Время начала резервного копирования"

#: ../src/ClientModeBox.py:532
msgid "Time of sync to start"
msgstr "Время начала синхронизации"

#: ../src/ClientModeBox.py:638
msgid "To apply new settings, you need to restart GUI your STB. Do you want restart it now?"
msgstr "Чтобы применить новые настройки, вам необходимо перезапустить графический интерфейс вашего ресивера. Вы хотите перезапустить его сейчас?"

#: ../src/MountManager.py:152
msgid "Type: "
msgstr "Тип: "

#: ../src/MountManager.py:226
msgid "Unmount"
msgstr "Размонтировать"

#: ../src/ImageManager.py:560
#, python-format
msgid ""
"Unzip error (also sent to any debug log):\n"
"%s"
msgstr ""
"Ошибка распаковки (также в журнале отладки):\n"
"%s"

#: ../src/MountManager.py:470
msgid "Updating mount locations..."
msgstr "Обновление точек монтирования..."

#: ../src/BackupManager.py:202
msgid "Use Mount Manager Please"
msgstr "Используйте диспетчер монтирования, пожалуйста"

#: ../src/ImageManager.py:347
msgid "Use Mount Manager and press green setup mounts"
msgstr "Используйте диспетчер монтирования и нажмите зеленую кнопку настройки монтирования."

#: ../src/MountManager.py:256
msgid "Use as HDD"
msgstr "Использовать как HDD"

#: ../src/ClientModeBox.py:528
msgid "Use remote HDD"
msgstr "Использовать удаленный жесткий диск"

#: ../src/ClientModeBox.py:529
msgid "Use remote timers"
msgstr "Используйте удаленные таймеры"

#: ../src/Multibootmgr.py:38
msgid "Use the cursor keys to select an installed image and then Erase button."
msgstr "Используйте клавиши курсора, чтобы выбрать установленное изображение, а затем кнопку удаления."

#: ../src/ui.py:62
msgid "Use this box as a client of a server."
msgstr "Используйте этот ресивер как клиент сервера."

#: ../src/BackupManager.py:174 ../src/BackupManager.py:316
#: ../src/BackupManager.py:367 ../src/ImageManager.py:299
#: ../src/ImageManager.py:445
msgid "View progress"
msgstr "Процесс выполнения"

#: ../src/BackupManager.py:115 ../src/plugin.py:206
msgid "Vision Backup Manager"
msgstr "Vision Менеджер резервного копирования"

#: ../src/BackupManager.py:912
msgid "Vision Backup Manager Setup"
msgstr "Установки Менеджера резервного копирования"

#: ../src/BackupManager.py:1027
msgid "Vision Backup manager logs"
msgstr "Логи Менеджера резервного копирования"

#: ../src/ClientModeBox.py:120 ../src/ClientModeBox.py:478
#: ../src/ClientModeBox.py:893 ../src/plugin.py:211
msgid "Vision Client Mode Box"
msgstr "Окно режима клиента"

#: ../src/ui.py:46 ../src/plugin.py:93 ../src/plugin.py:193
msgid "Vision Core"
msgstr "Vision Ядро"

#: ../src/H9SDmanager.py:34
msgid "Vision H9 SDcard Manager"
msgstr "Zgemma H9 SD-карт менеджер"

#: ../src/ImageManager.py:241 ../src/plugin.py:207
msgid "Vision Image Manager"
msgstr "Vision имидж менеджер"

#: ../src/ImageManager.py:129
msgid "Vision Image Manager Setup"
msgstr "Установки имидж менеджера"

#: ../src/MountManager.py:216 ../src/plugin.py:208
msgid "Vision Mount Manager"
msgstr "Менеджер монтирования"

#: ../src/Multibootmgr.py:36
msgid "Vision Multiboot Image Manager"
msgstr "Мультизагрузка"

#: ../src/IPKInstaller.py:40
msgid "Vision Opkg Install"
msgstr "Vision установщик"

#: ../src/plugin.py:210
msgid "Vision SWAP Manager"
msgstr "Управление файлом подкачки"

#: ../src/ScriptRunner.py:50 ../src/plugin.py:209
msgid "Vision Script Runner"
msgstr "Запуск скриптов"

#: ../src/plugin.py:198
msgid "Vision Script runner"
msgstr "Запуск скриптов"

#: ../src/SoftcamManager.py:86 ../src/plugin.py:112 ../src/plugin.py:196
msgid "Vision Softcam"
msgstr "Эмуляторы"

#: ../src/SoftcamManager.py:571
msgid "Vision Softcam logs"
msgstr "Логи эмулятора"

#: ../src/SoftcamManager.py:326
msgid "Vision Softcam setup"
msgstr "Настройка эмулятора"

#: ../src/SwapManager.py:112
msgid "Vision Swap Manager"
msgstr "Менеджер файла подкачки"

#: ../src/SwapManager.py:332
msgid "Wait please while creating swap file..."
msgstr "Пожалуйста, подождите, пока создается файл подкачки..."

#: ../src/SoftcamManager.py:178
msgid ""
"Wait please while scanning\n"
"for softcam's..."
msgstr ""
"Пожалуйста, подождите, пока идет поиск\n"
"эмулятора..."

#: ../src/SwapManager.py:157
msgid "Wait please while scanning..."
msgstr "Пожалуйста, подождите, пока идет сканирования..."

#: ../src/MountManager.py:328
msgid "Warning, this device is used for more than one mount point!\n"
msgstr "Внимание, это устройство используется более чем для одной точки монтирования!\n"

#: ../src/BackupManager.py:61 ../src/ImageManager.py:73
#: ../src/ClientModeBox.py:52
msgid "Weekly"
msgstr "Еженедельно"

#: ../src/RestoreWizard.py:106 ../src/RestoreWizard.py:109
msgid "Yes, to restore plugins"
msgstr "Да, чтобы восстановить плагины"

#: ../src/RestoreWizard.py:100
msgid "Yes, to restore settings"
msgstr "Да, чтобы восстановить настройки"

#: ../src/ImageManager.py:530
msgid "You have decided not to flash image."
msgstr "Вы отказались прошивать имидж."

#: ../src/SwapManager.py:352
msgid "You have to create a swap file before trying to activate the autostart."
msgstr "Вам нужно создать файл подкачки, прежде чем активировать автозапуск."

#: ../src/MountManager.py:361
#, python-format
msgid ""
"You may have to press red button again.\n"
"Use %s as HDD ?"
msgstr ""
"Возможно, вам придется снова нажать красную кнопку.\n"
"Использовать %s в качестве жесткого диска?"

#: ../src/RestoreWizard.py:262
msgid "Your receiver could not connect to the plugin feeds at this time. Please try using the Backup manager to restore plugins later."
msgstr "Ваш %s %s не может подключиться к фиду. Для восстановления плагинов, повторите попытку позже."

#: ../src/ImageManager.py:716
msgid ""
"Your receiver is about to create a full image backup, this can take about 6 minutes to complete.\n"
"Do you want to allow this?"
msgstr ""
"Ваш %s %s создает полную резервную копию имиджа, это может занять около 6 минут.\n"
"Продолжить?"

#: ../src/BackupManager.py:1139
msgid ""
"Your receiver is about to run a backup of your settings and to detect your plugins.\n"
"Do you want to allow this?"
msgstr ""
"Ваш %s %s запускает резервное копирование настроек и плагинов.\n"
"Продолжить?"

#: ../src/BackupManager.py:540
msgid "Your receiver is not connected to a network. Please check your network settings and try again."
msgstr "Ваш %s %s не подключен к сети. Пожалуйста, проверьте настройки сети и попробуйте снова."

#: ../src/RestoreWizard.py:258
msgid "Your receiver is not connected to a network. Please try using the Backup manager to restore plugins later when a network connection is available."
msgstr "Ваш %s %s не подключен к сети. Для восстановления плагинов, повторите попытку позже, когда будет доступно сетевое соединение."

#: ../src/BackupManager.py:558
msgid "Your receiver is not connected to the Internet. Please check your network settings and try again."
msgstr "Ваш %s %s не подключен к Интернету. Пожалуйста, проверьте настройки сети и попробуйте снова."

#: ../src/RestoreWizard.py:158
msgid "Your receiver is not connected to the Internet. Please try using Backup manager to restore plugins later."
msgstr "Ваш %s %s не подключен к Интернету. Для восстановления плагинов, повторите попытку позже."

#: ../src/RestoreWizard.py:266
msgid "Your receiver is not connected to the Internet. Please try using the Backup manager to restore plugins later."
msgstr "Ваш %s %s не подключен к Интернету. Для восстановления плагинов, повторите попытку позже."

#: ../src/ui.py:54 ../src/MountManager.py:222 ../src/MountManager.py:434
#: ../src/ScriptRunner.py:56 ../src/BackupManager.py:121
#: ../src/BackupManager.py:818 ../src/BackupManager.py:918
#: ../src/BackupManager.py:1044 ../src/ImageManager.py:135
#: ../src/ImageManager.py:255 ../src/ImageManager.py:1455
#: ../src/SoftcamManager.py:92 ../src/SoftcamManager.py:332
#: ../src/SoftcamManager.py:407 ../src/SoftcamManager.py:492
#: ../src/SoftcamManager.py:577 ../src/IPKInstaller.py:46
#: ../src/SwapManager.py:118 ../src/ClientModeBox.py:127
#: ../src/ClientModeBox.py:484 ../src/ClientModeBox.py:891
msgid "https://github.com/OpenVisionE2"
msgstr ""

#: ../src/ui.py:52 ../src/MountManager.py:220 ../src/MountManager.py:432
#: ../src/ScriptRunner.py:54 ../src/BackupManager.py:119
#: ../src/BackupManager.py:816 ../src/BackupManager.py:916
#: ../src/BackupManager.py:1042 ../src/ImageManager.py:133
#: ../src/ImageManager.py:253 ../src/ImageManager.py:1453
#: ../src/SoftcamManager.py:90 ../src/SoftcamManager.py:330
#: ../src/SoftcamManager.py:405 ../src/SoftcamManager.py:490
#: ../src/SoftcamManager.py:575 ../src/IPKInstaller.py:44
#: ../src/SwapManager.py:116 ../src/ClientModeBox.py:125
#: ../src/ClientModeBox.py:482 ../src/ClientModeBox.py:889
msgid "https://openvision.tech"
msgstr ""

#: ../src/ImageManager.py:613
#, python-format
msgid ""
"ofgwrite error (also sent to any debug log):\n"
"%s"
msgstr ""
"Ошибка ofgwrite (также в журнале отладки):\n"
"%s"

#: ../src/ImageManager.py:493 ../src/ImageManager.py:512
#, python-format
msgid "slot%s - %s"
msgstr "Слот%s - %s"

#: ../src/Multibootmgr.py:74
#, python-format
msgid "slot%s - %s "
msgstr "Слот%s - %s "

#: ../src/ImageManager.py:493 ../src/ImageManager.py:512
#, python-format
msgid "slot%s - %s (current image)"
msgstr "Слот%s - %s (текущий имидж)"

#: ../src/MountManager.py:119 ../src/MountManager.py:132
msgid "unavailable"
msgstr "Недоступен"
